pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+python_files = test_*.py
+# For the benefit of test_wasyncore.py
```

Comment:
Not sure you need this... also instead of a new file, drop this into `pyproject.toml`, see https://github.com/Pylons/pyramid_mako/blob/main/pyproject.toml#L27

Reply:
I removed the extra config from the file but I'm leaving the file in for now because it's consistent with the other repos I'm maintaining until I do another pass on that.